src/naf-dialog-adapter.js

```diff
@@ -613,141 +582,197 @@ export default class DialogAdapter {
     // Not implemented
   }
 
-  async _joinRoom() {
-    debug("_joinRoom()");
-
-    try {
-      this._mediasoupDevice = new mediasoupClient.Device({});
+  async createSendTransport(iceServers) {
```

Comment:
Didn't comb through the changes here very closely since I think this is just moving code around, let me know if that is not the case.